frontend_vue/src/utils/confettiEffect.ts

```diff
@@ -25,8 +26,8 @@ export function launchConfetti() {
   myConfetti({
     particleCount: 100,
     spread: 160,
-    origin: { y: 0.9 },
-    colors: ['#F2059F', '#04D9B2', '#80C7F2'],
+    origin: { y: .4 },
+    colors: TOKEN_COLOR_PALETTES[token_type],
```

Comment:
I guess we should add a fallback, in case no colour palette is provided

Reply:
Sure, makes sense